features/step_definitions/challenge_steps.rb

```diff
@@ -68,6 +68,14 @@
     step %{a canonical fandom "Unrequested"}
 end
 
+Given /^I have overlapping challenge tags set ?up$/ do 
+  step "I have standard challenge tags setup"
+  step %{I add the relationship tags "Sam Carter/Daniel Jackson, Daniel Jackson/Jack O'Neill, Sam Carter/Daniel Jackson/Jack O'Neill" to the tag set "Standard Challenge Tags"}
```

Comment:
Do you think it would make sense to have a separate tag set to distinguish these from the standard tags; maybe "Overlapping Challenge Tags"?